pyht/async_client.py

```diff
@@ -446,34 +447,48 @@ async def _tts_ws(
         assert self._inference_coordinates is not None, "No connection"
         metrics.append("text", str(text)).append("endpoint",
                                                  str(self._inference_coordinates[voice_engine]["websocket_url"]))
-        request_id = str(uuid.uuid4())
         json_data = http_prepare_dict(text, options, voice_engine)
-        json_data["request_id"] = request_id
 
         for attempt in range(1, self._max_attempts + 1):
             try:
                 assert self._inference_coordinates is not None, "No connection"
                 ws_address = self._inference_coordinates[voice_engine]["websocket_url"]
                 if self._ws is None:
                     self._ws = await connect(ws_address)
+                    self._ws_requests_sent = 0
+                    self._ws_responses_received = 0
                 try:
                     await self._ws.send(json.dumps(json_data))
+                    self._ws_requests_sent += 1
                 except ConnectionClosed as e:
                     logging.debug(f"Reconnecting websocket which closed unexpectedly: {e}")
                     self._ws = await connect(ws_address)
+                    self._ws_requests_sent = 0
+                    self._ws_responses_received = 0
                     await self._ws.send(json.dumps(json_data))
+                    self._ws_requests_sent += 1
```

Comment:
Some redundancy here
```suggestion
                    self._ws_requests_sent = 1
                    self._ws_responses_received = 0
                    await self._ws.send(json.dumps(json_data))
```